scripts/README-wmp-check.md

```diff
@@ -0,0 +1,165 @@
+# [wmp-check.sh](./wmp-check.sh)
+> This script originally developed in repo https://github.com/scmschmidt/wmp_check.git
+> Merged into this repo at v1.1.1
+
+This script checks the setup of Workload Memory Protection.
+
+- Correct setup of cgroup2
+- Checc the required rpm packages version
+- Ownership and permission of the capture program
+- WMP entries of SAP instance profiles
+- Correct cgroup of running SAP instance processes.
+- Correct setup of SAP.slice
+- Sane configuration of MemoryLow (It can not determine, if the MemoryLow value has been chosen wisely!)
+- Setup of the optional memory sampler
+- Setup of optional swap accounting
+
+Please keep in mind:
+
+ - It does not check if you have the latest version installed, only minimum version.
+ - It assumes SAP instances profiles can be found beneath /usr/sap/<SID>/SYS/profile/.
+ - This tool does not check, if the memory.low value is set correctly.
+
+
+## Usage
+```
+wmp-check.sh
```

Comment:
This usage assume the script is in `$PATH`. Perhaps I wouldn't put the script to `$PATH` but instead referenced to it with full path.